packages/help-center/src/components/help-center-support-chat-message.tsx

```diff
@@ -108,11 +108,7 @@ export const HelpCenterSupportChatMessage = ( {
 						}
 					/>
 					<span>
-						{ getRelativeTimeString( {
-							timestamp: received * 1000,
-							locale,
-							style: 'long',
-						} ) }
```

Comment:
Just curious, what was the issue with this one?

Reply:
![image](https://github.com/user-attachments/assets/2520f4d7-fbce-4bae-8e6a-7ef056f9b75a)